src/lsp/cobol_lsp/lsp_request.ml

```diff
@@ -168,16 +181,7 @@ let handle_range_formatting registry params =
       end_line = end_.line + 1
     }
   in
-  (*the range must contain the whole lines*)
-  let range =
-    (*TODO:find a simple method to get the number of letters of one line
-           (the code below use 1000 as the upperbound)                   *)
-    Range.{
-      start = Position.create ~line:start.line ~character:0;
-      end_ = Position.create ~line:end_.line ~character:1000;
-    }
-  in
-  let newText =
+  let editList =
```

Comment:
I just kept the local style (`newText` was named this way because it is passed to `TextEdit.create` which expects a `newText` named parameter, I believe). No issues with using snake case (but this is O**Caml**!).